### PR TITLE
Create profile controller

### PR DESCRIPTION
**Pull request by GurkirtSingh**
### What this PR does (required):
- creates controllers for creating, updating or getting profiles.
- Includes routes as well.

Link to issue https://hatchways.notion.site/Profile-Controller-d0246f2e1bfd4c94b4a8b5467f14a995

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -11,13 +11,21 @@ const logger = require("morgan");
 
 const authRouter = require("./routes/auth");
 const userRouter = require("./routes/user");
+const profileRouter = require("./routes/profile");
+
+require("dotenv").config();
```
Comment (moffatethan) [inline server/app.js]: Could this be moved to the top of the file?

File: server/models/Profile.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ const profileSchema = new mongoose.Schema({
   location: {
     city: {
       type: String,
+      index: true,
```
Comment (moffatethan) [inline server/models/Profile.js:36]: 💯

**Review by moffatethan — APPROVED** [review]
LGTM! Just those minor changes then you can merge in. Thanks!

File: server/app.js (hunk 1 above)
Comment (GurkirtSingh) [reply server/app.js]: Absolutely

**Pull request merged**